docs/dev_guide/diagnostic_ref/dpct1085.rst

```diff
@@ -26,45 +26,95 @@ For example, this original CUDA\* code:
 .. code-block:: cpp
    :linenos:
   
-     _global_ void kernel() {
-       int Input, Output1, Output2, Lane;
-       ...
-  
-       // original code logic
-       Output1 = __shfl(Input, Lane, 32);
-       Output2 = __shfl_xor(Input, Lane, 16);
-     }
+    __global__ void kernel(int* data1, int* data2) {
+      typedef cub::WarpScan<int> WarpScan;
+      typedef cub::WarpScan<int, 16> WarpScan16;
+    
+      typename WarpScan::TempStorage temp1;
+      typename WarpScan16::TempStorage temp2;
+    
+      int input = data1[threadIdx.x];
+      int output1 = 0;
+      int output2 = 0;
+      WarpScan(temp1).InclusiveSum(input, output1);
+      data1[threadIdx.x] = output1;
+      WarpScan16(temp2).InclusiveSum(input, output2);
+      data2[threadIdx.x] = output1;
+    }
+    
+    void foo(int* data1, int* data2) {
+      kernel<<<1, 32>>>(data1, data2);
+    }
 
 results in the following migrated SYCL\* code:
 
 .. code-block:: cpp
    :linenos:
   
-     void kernel(int WarpSize, sycl::nd_item<3> item_ct1) {
-       int Input, Output1, Output2, Lane;
-       ...
-  
-       // original code logic
-       Output1 = Item_ct1.get_sub_group().shuffle(Input, Lane, 32);
-       /* DPCT1085 */
-       Output2 = Item_ct1.get_sub_group().shuffle_xor(Input, Lane, 16);
-     }
+    void kernel(int* data1, int* data2, const sycl::nd_item<3> &item_ct1) {
+    
+      int input = data1[item_ct1.get_local_id(2)];
+      int output1 = 0;
+      int output2 = 0;
+      output1 = sycl::inclusive_scan_over_group(item_ct1.get_sub_group(), input,
+                                                sycl::plus<>());
+      data1[item_ct1.get_local_id(2)] = output1;
+      /*
+      DPCT1085:0: The function inclusive_scan_over_group requires sub-group size to
+      be 16, while other sub-group functions in the same SYCL kernel require a
+      different sub-group size. You may need to adjust the code.
+      */
+      output2 = sycl::inclusive_scan_over_group(item_ct1.get_sub_group(), input,

```

Comment:
Could you file a JIRA to followup:  migrate WarpScan16(temp2).InclusiveSum(input, output2) with sub-group size 32?